scss/elements/_forms.scss

```diff
@@ -240,3 +240,48 @@ button {
     opacity: 0.5;
   }
 }
+
+// Homepage candidate/commitee search box styles
+
+.slab--primary .grid__item .js-search-type {
```

Comment:
Thanks for clarifying @jenniferthibault. I removed the additional styles and the search section is now defaulting to the original styles.